src/components/basic_table/in_memory_table.tsx

```diff
@@ -678,6 +683,8 @@ export class EuiInMemoryTable<T> extends Component<
     return (
       <div>
         {searchBar}
+        {childrenBetween && <EuiSpacer size="l" />}
```

Comment:
```suggestion
        {childrenBetween != null <EuiSpacer size="l" /> : null}
```

These kind of `A && B` content inclusions with React are weird due to how JS processes the `&&` and what React will render: `contentBetween={0}` renders _00_ with no spacer. Testing the value against `null`+`undefined` better aligns the logic with React's rendering.

Reply:
Thank you for the suggestion. Fixed  🎸